imports/client/components/UnauthenticatedRoute.tsx

```diff
@@ -50,7 +49,7 @@ const UnauthWrapper = React.memo((props: UnauthWrapperProps) => {
   }
 
   if (tracker.userId) {
-    const state = _.extend({ pathname: '/', search: undefined }, props.location.state);
+    const state = { pathname: '/', search: undefined, ...(props.location.state as any) };
```

Comment:
FWIW this (and other `as any` casts that I added) aren't actually any worse than what we had before, but the `_.extend` or `_.omit` were masking it.